### DISCUSSION TITLE
Show Player in Menus

### DISCUSSION THREAD

**Discussion by vrnord** [alandtse/skyrim_vr_address_library#29] [Ideas]
Any chance we could get a VR port of [Show Player in Menus](https://www.nexusmods.com/skyrimspecialedition/mods/81291?tab=description)?  It uses CommonLibSSE-NG and the source is posted [here](https://github.com/derickso/ShowPlayerInMenus).  

It would be a more natural version of selfie mode in VRIK or the old 'VR Mirror" mod that would allow you to see your player avatar while equipping armour/weapons etc.

Thanks for the consideration and for all the effort you put into making Skyrim the best VR experience!

**Comment by DJLegends1011** [comment]
would be nice tbh
